test/test_readcalls.c

```diff
@@ -195,7 +195,7 @@ void test_qso_array_init(void **state) {
 
 void test_readcalls_simple_log(void **state) {
     int lines;
-    gchar *qso_line = g_strndup(QSO1, LOGLINELEN - 1);
+    gchar *qso_line = g_strndup(QSO1, 87);
```

Comment:
Any reason to change this and line 214?

Reply:
WIthout this lengths of logline and note is not equal. I had to change that after switching to getline what causes buffer_len is setup by getline.

Reply:
restored original version